gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -1044,7 +1056,7 @@ public void testGlobStatus() throws IOException {
     createFile(new Path("/directory1/subdirectory2/file1"), data);
     createFile(new Path("/directory1/subdirectory2/file2"), data);
 
-    FileStatus[] rootDirectories = ghfs.globStatus(new Path("/d*"));
+    FileStatus[] rootDirectories = ghfs.globStatus(new Path("/directory1*"));
```

Comment:
Why do we need this change?

Reply:
We have changed bucket name from `gcsio-test` --> `dataproc-gcsio-test`.

So, earlier `d*` would only provide `directory1` whcich was created in test. But with bucket name cchange `d*` would provide two objects `directory1` and `dataproc-gcsio-test..`. Which is causing test to fail.

------

Why a change in bucket name is affecting the directories inside the bucket?

* Test class [`GoogleHadoopFileSystemIntegrationTest`](https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java#L111) where we create a bucket using URI. We not only create the bucket but also endup creating an object with same name as bucketName. and this becomes an issue.